NeffCodes/retinopathy-risk-assessment-tool
# Resolves #53 - Add tests to verify retina scan deletion

This PR adds some tests to ensure the retina scan is properly handled when deleted from our application. It also adds new dependencies pytest and pytest-django

### To Test

- Start your virtual environment
- run `pip install -r requirements.txt` to add new dependencies
- if you are not already, cd into the 'rrat' directory
- run `pytest retina_photos/` and verify all tests pass
